firedrake/slate/slate.py

```diff
@@ -548,39 +592,23 @@ def __init__(self, tensor, coefficient):
             "Argument function space must be the same as the "
             "coefficient function space."
         )
-        super(Action, self).__init__()
-        self.tensor = tensor
-        self._acting_coefficient = coefficient
+        super(Action, self).__init__(tensor,
+                                     assembled_coeffs=(coefficient,))
+        self.actee = coefficient
```

Comment:
You have a double accounting of coefficient. In fact, this class is the only one (unless I missaw something) which actually uses `assembled_coeffs` which suggests that it shouldn't exist on the parent class.